experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -77,6 +77,24 @@ class Flowables {
         std::forward<OnSubscribe>(function)));
   }
 
+  template <typename T>
+  static Reference<Flowable<T>> empty() {
+    auto lambda = [](Subscriber<T>& subscriber, int64_t) {
+      subscriber.onComplete();
+      return std::make_tuple(static_cast<int64_t>(0), true);
```

Comment:
You're worried about the case where zero doesn't fit in an int?

Besides, this matches the existing style within this file.